lbz/authz/decorators.py

```diff
@@ -29,7 +29,8 @@ def check_permission(resource: Resource, permission_name: str) -> dict:
         policy_override=guest_authorization_policy,
     )
     authorizer.check_access()
-    return authorizer.restrictions
+    restrictions: dict = authorizer.restrictions
```

Comment:
The same thing as before, there is no point to create a separate variable named `restrictions` if we want to not use it and just return one line below. The previous version was fully correct and nicer 😉